### ISSUE TITLE
Suggestion: use of package.json instead of index.js

### ISSUE THREAD

**Issue by bradder555** [joshgeller/react-redux-jwt-auth-example#21]
Hi All

I'd like to propose the use of package.json instead of index.js (i've already done this on my copy of the project.)

I found it was confusing having four files open all called index.js

so for example by renaming to containers.js and creating the file package.json with
`
{
  "name": "containers",
  "version": "0.0.0",
  "private": true,
  "main": "./containers.js"
}
`

It's far less confusing.

Other tweaks for clarity is using the naming convention "<<Dir>>Title", i.e.
ViewHome, ViewLogin...
ReducerAuth, ReducerData...

**Comment by mars76** [comment]
I agree with you 100%. Not sure why we need to have index.js in each of the folders.

If you don't my could you publish your version.

**Comment by Zhouzi** [comment]
Actually `index.js` is an "entry point" that's loaded when you require a directory without specifying a file. So assuming this folder structure:

```
reducers/
  auth.js
  index.js
  posts.js
```

You can `require('./reducers')` to load `./reducers/index.js` which could `combineReducers()` for example. That's a JavaScript convention/specification.
